src/platform_impl/windows/event_loop.rs

```diff
@@ -1652,14 +1682,14 @@ unsafe fn public_window_callback_inner<T: 'static>(
 
     // Only sent on Windows 8.1 or newer. On Windows 7 and older user has to log out to change
     // DPI, therefore all applications are closed while DPI is changing.
-    winuser::WM_DPICHANGED => {
+    win32wm::WM_DPICHANGED => {
       use crate::event::WindowEvent::ScaleFactorChanged;
 
       // This message actually provides two DPI values - x and y. However MSDN says that
       // "you only need to use either the X-axis or the Y-axis value when scaling your
       // application since they are the same".
       // https://msdn.microsoft.com/en-us/library/windows/desktop/dn312083(v=vs.85).aspx
-      let new_dpi_x = u32::from(LOWORD(wparam as DWORD));
+      let new_dpi_x = util::to_loword(wparam.0 as u32) as u32;
```

Comment:
again `from_loword`? I am a bit confused about these conversions, that's why I wanted them into a util functions so the name can clear any confusion.